src/app/screens/Onboard/ConnectLndHub/index.jsx

```diff
@@ -78,13 +80,28 @@ export default function ConnectLndHub() {
                   type="text"
                   required
                   placeholder="lndhub://..."
+                  value={formData.uri}
                   onChange={handleChange}
                 />
                 <small>
                   BlueWallet users find the Backup/Export URI in the lightning
                   wallet settings.
                 </small>
               </div>
+              <p className="text-center my-4">OR</p>
+              <QrcodeScanner
+                fps={10}
+                qrbox={250}
+                qrCodeSuccessCallback={(decodedText) => {
```

Comment:
maybe we can "stop" the scanning after we successfully have done a scan and set the form data?